Makefile

```diff
@@ -29,8 +29,8 @@ test: deps
 
 # build will ensure all of our tests pass and then build the go binary
 build: test
-	go build -mod=vendor ./...
+	$GOPROXY go build -mod=vendor ./...
```

Comment:
I am not sure what does it means to just add `$GOPROXY` at the start of the `go build` command without actually setting a value for it.

**Note:** If GOPROXY is unset, empty or set to direct then go get will use a direct connection to the VCS (e.g github.com) Ref: https://arslan.io/2019/08/02/why-you-should-use-a-go-module-proxy/

Also, `go build  mod=vendor ./...` doesn't download any dependencies, it just looks for dependencies to build the binary, in the `vendor` folder. If the dependencies don't exist, it will fail. So, setting the `GOPROXY` var for this command doesn't really make sense.

I believe the right approach here is to set the `GOPROXY` env var to `"https://proxy.golang.org"` before running `go mod vendor` which is the command that will actually install all the dependencies.

Reply:
Hey @mansinahar , indeed, you are right, I just pushed the changes ! Thanks :)